texar/modules/classifiers/bert_classifiers.py

```diff
@@ -100,36 +100,39 @@ def __init__(self,
                 self._logits_layer = nn.Linear(self._hparams.hidden_size,
                                                self.num_classes,
                                                **logit_kwargs)
-        self.is_binary = None
+
+        self.is_binary = (self.num_classes == 1) or \
+                         (self.num_classes <= 0 and
+                          self._hparams.encoder.dim == 1)
 
     @staticmethod
     def default_hparams():
         r"""Returns a dictionary of hyperparameters with default values.
 
         .. code-block:: python
 
-        {
+            {
             # (1) Same hyperparameters as in BertEncoder
             ...
             # (2) Additional hyperparameters
-            `num_classes`: 2,
-            `logit_layer_kwargs`: None,
-            `clas_strategy`: `cls_time`,
-            `max_seq_length`: None,
-            `dropout`: 0.1,
-            `name`: `bert_classifier`
+            "num_classes": 2,
+            "logit_layer_kwargs": None,
+            "clas_strategy": "cls_time",
+            "max_seq_length": None,
+            "dropout": 0.1,
+            "name": "bert_classifier"
             }
 
         Here:
 
         1. Same hyperparameters as in
-        :class:`~texar.modules.BertEncoder`.
-        See the :meth:`~texar.modules.BertEncoder.default_hparams`.
-        An instance of BertEncoder is created for feature extraction.
+            :class:`~texar.modules.BertEncoder`.
```

Comment:
Following lines in a list should indent to the beginning of the first line, in this case, right after `1. `. So these lines should be indented with 3 spaces, not 4.

Reply:
Sorry for all the docs-related error. I should have checked the generated `html` before submitting the PR.